pkg/common/encoding/java_type/java_type.go

```diff
@@ -0,0 +1,119 @@
+package java_type
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import (
+	"reflect"
+	"time"
+)
+
+import (
+	hessian "github.com/apache/dubbo-go-hessian2"
+)
+
+// GetArgType is copied from hessian, to get java class type of interface
+func GetArgType(v interface{}) string {
```

Comment:
I am thinking here should call hessian API instead of copying & pasting. You may submit a PR for hessian2 to wrap an exported method.

Reply:
If this function is updated by hessian2, it will cause some problems unexpectedly.

Reply:
This file comes from a previous modification to grpc-go that was tested in dubbo-go-example.

Reply:
I am saying that If this function is updated by hessian2 in the future, you may refer to https://github.com/apache/dubbo-go-hessian2/blob/master/request.go. IMO, you should submit a PR to export the `getArgType` method. After the PR is merged, bump the version and call it directly.